### PR TITLE
 migrated to v1alpha2

### PR DESCRIPTION
**Pull request by xrajesh**
**Related Issue:**  open-cluster-management/backlog#<ISSUE_NUMBER>

### CODE REVIEW

File: pkg/transforms/policyreport.go [hunk 1]
```diff
@@ -5,35 +5,36 @@ package transforms
...
-	Data ReportData `json:"data"`
+	Policy      string     `json:"policy"`
+	Description string     `json:"message"`
+	Category    string     `json:"category"`
+	Result      string     `json:"result"`
+	Properties  ReportData `json:"properties"`
 }
 
 // ReportData rule violation data
 type ReportData struct {
```
Comment (zlayne) [inline pkg/transforms/policyreport.go:29]: Maybe rename to ReportProperties?

File: pkg/transforms/policyreport.go [hunk 2]
```diff
@@ -5,35 +5,36 @@ package transforms
...
-	Policy string `json:"policy"`
-	Message string `json:"message"`
-	Category string `json:"category"`
-	Status string `json:"status"`
-	Data ReportData `json:"data"`
+	Policy      string     `json:"policy"`
+	Description string     `json:"message"`
+	Category    string     `json:"category"`
+	Result      string     `json:"result"`
+	Properties  ReportData `json:"properties"`
```
Comment (zlayne) [inline pkg/transforms/policyreport.go:25]: Should we add timestamp here as well?

File: pkg/transforms/policyreport.go [hunk 3]
```diff
@@ -48,9 +49,9 @@ func PolicyReportResourceBuilder(pr *PolicyReport) *PolicyReportResource {
 	node.Properties["apigroup"] = gvk.Group
 
 	// Extract the properties specific to this type
-	node.Properties["message"] = string(pr.Results[0].Message)
+	node.Properties["message"] = string(pr.Results[0].Description)
```
Comment (zlayne) [inline pkg/transforms/policyreport.go:52]: Would it be less confusing to use description as the property here?

**Review by zlayne — APPROVED** [review]

File: pkg/transforms/policyreport.go (hunk 2 above)
Comment (xrajesh) [reply pkg/transforms/policyreport.go:25]: Not sure we want Timestamp in search , until we need it we dont want to cache in redis .

**Pull request merged**